modules/data.land/R/cohort2pool.R

```diff
@@ -30,27 +30,63 @@ cohort2pool <- function(veg_file, allom_param = NULL) {
   
   ## Grab allometry
   if(is.null(allom_param)){
-    a <- 2                        
-    b <- 0.3
+    a <- -2                        
+    b <- 2.5
   } else {
     print("user provided allometry parameters not yet supported")
     return(NULL)
   }
   
-  #Calculate AGB
+  #Calculate Total Biomass
+  #Need to Convert the total kg of biomass to (kg/m^2) but data on area of land does not exist
+  #In it's place,dividing by number of trees.
```

Comment:
This is not a sound assumption. The area of the land needs to be passed in explicitly and should be part of the veg workflow already.

Reply:
Also not clear why you switched from AGB to total biomass? There's going to be a lot less constraint on the total biomass allometries (much greater uncertainty) because of the rarity of belowground data